src/go/plugin/go.d/collector/mysql/charts.go

```diff
@@ -364,6 +367,42 @@ var (
 			{ID: "innodb_log_writes", Name: "writes", Algo: module.Incremental, Mul: -1},
 		},
 	}
+	chartInnoDBLogOccupancy = module.Chart{
+		ID:       "innodb_log_occupancy",
+		Title:    "InnoDB Redo Log Occupancy",
+		Units:    "percentage",
+		Fam:      "innodb",
+		Ctx:      "mysql.innodb_log_occupancy",
+		Type:     module.Area,
+		Priority: prioInnoDBLog,
+		Dims: module.Dims{
+			{ID: "innodb_log_occupancy", Name: "occupancy", Algo: module.Absolute, Div: 1000},
+		},
+	}
+	chartInnoDBLogProgress = module.Chart{
+		ID:       "innodb_log_progress",
+		Title:    "InnoDB Log Sequence Progress",
+		Units:    "MiB/s",
+		Fam:      "innodb",
+		Ctx:      "mysql.innodb_log_progress",
+		Type:     module.Area,
+		Priority: prioInnoDBLog,
+		Dims: module.Dims{
+			{ID: "innodb_log_sequence_number", Name: "sequence number", Algo: module.Incremental, Div: 1024 * 1024},
+			{ID: "innodb_last_checkpoint_at", Name: "last checkpoint", Algo: module.Incremental, Mul: -1, Div: 1024 * 1024},
+		},
+	}
+	chartInnoDBCheckpointAge = module.Chart{
+		ID:       "innodb_checkpoint_age",
+		Title:    "InnoDB Checkpoint Age",
+		Units:    "MiB",
+		Fam:      "innodb",
+		Ctx:      "mysql.innodb_checkpoint_age",
+		Priority: prioInnoDBLog,
+		Dims: module.Dims{
+			{ID: "innodb_checkpoint_age", Name: "age", Algo: module.Absolute, Div: 1024 * 1024},
+		},
+	}
```

Comment:
1. Keep the units in bytes (B) instead of converting to MiB.

2. Can you clarify what checkpoint age represents exactly? Is it a size in bytes or a time/duration metric? The name suggests it might not belong in bytes, so I want to confirm it’s the correct unit.